### ISSUE TITLE
You have requested a non-existent service "templating.helper.assets".

### ISSUE THREAD

**Issue by bvisonl** [liip/LiipMonitorBundle#125]
I understand that templating.helper.assets was removed in SF2 3.0. Is there a fix to this?

**Comment by lsmith77** [comment]
not yet .. PRs welcome

**Comment by nicoschoenmaker** [comment]
There is a workaround, to add PHP as a rendering engine in `config.yml`.

```
framework:
    templating:
        engines: [twig, php]
```

But this is a bit of a hidden requirement of this bundle right now, would be better to fix it somehow.

**Comment by arnovr** [comment]
This does not work for me with SF3.

**Comment by vchampion** [comment]
So, while PR is still waited it's possible to define aliases for needed services and all works well. So, add to global services.yml
```
services:
    templating.helper.assets: '@assets.packages'
    templating.helper.router: '@router'
```

**Comment by kbond** [comment]
fixed in [2.5.2](https://github.com/liip/LiipMonitorBundle/releases/tag/2.5.2)
